src/IterableInboxMessageList.tsx

```diff
@@ -52,9 +52,7 @@ export const IterableInboxMessageList = ({
         swipingCheck={setSwiping}
         messageListItemLayout={messageListItemLayout}
         deleteRow={(messageId: string) => deleteRow(messageId)}
-        handleMessageSelect={(messageId: string, i: number) =>
-          handleMessageSelect(messageId, i)
-        }
+        handleMessageSelect={handleMessageSelect}
```

Comment:
Whats the reason for this change?

Reply:
Cleanliness/readability.

Reply:
Gotcha. Can the same be done for deleteRow?